AsyncDisplayKit/AsyncDisplayKit-Prefix.pch

```diff
@@ -22,7 +22,3 @@
 #ifndef IG_LIST_KIT
   #define IG_LIST_KIT __has_include(<IGListKit/IGListKit.h>)
 #endif
-
-#ifndef YOGA
-  #define YOGA __has_include(<Yoga/Yoga.h>)
```

Comment:
Because the .pch was dropped in upstream for a while, I ported off of it to use ASAvailability instead.  If we want to port the other things as well, and find a way to make ASAvailability really widely included (ideally some kind of project level default), then we could remove the .pch (cc @Adlai-Holler)